lib/tasks/impossible_overdue_appointments.rake

```diff
@@ -0,0 +1,47 @@
+namespace :impossible_overdue_appointments do
+  desc 'Set all appointments with a newer blood pressure reading than the created_at date to "visited"'
+  task :fix_impossible_overdue_appointments, [:user_id] => :environment do |_t, args|
```

Comment:
```suggestion
  task :fix_scheduled_appointments_older_than_latest_blood_pressure, [:user_id] => :environment do |_t, args|
```

Reply:
Updated task name to `fix_impossible_overdue_appointments`.